### PR TITLE
DEL-8552: Del library page setup

### PR DESCRIPTION
**Pull request by RoshanShrestha123**
## Description

Del library page setup.

for local testing URL: http://localhost:8000/ 

## JIRA Link

https://lftechnology.atlassian.net/browse/DEL-8552

## Summary of Changes

- [x] Added the del library list page (as a landing page)
- [x] Header component added

## Additional implementation details (OPTIONAL)

- [x] Landing page remove (& marked as deprecated)

### After
image

### Before
image

### CODE REVIEW

File: src/views/del-library-page/del-library-page.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { LitElement, html, css } from 'lit-element';
+
+import '../../components/commons/header-component';
```
Comment (NadeemShakya) [inline src/views/del-library-page/del-library-page.js:3]: Leave a line here to group common imports.

File: src/views/del-library-page/del-library-page.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { DEL_LIBRARY_PAGE } from '../../constants/lang';
+
+/**
+ * Del library list page.
```
Comment (NadeemShakya) [inline src/views/del-library-page/del-library-page.js:7]: Add descriptive doc blocks.

File: src/views/del-library-page/del-library-page.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ */
+class DelLibraryPage extends LitElement {
+  /**
+   * Styles.
```
Comment (NadeemShakya) [inline src/views/del-library-page/del-library-page.js:11]: .

File: src/views/del-library-page/del-library-page.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+   */
+  static get styles() {
+    return css`
+      .page-wrapper {
+        background-color: var(--color-grey-100);
+        height: calc(100vh - 49px - var(--padding-40));
+        padding: var(--padding-20);
+        display: flex;
+        flex-direction: column;
+      }
```
Comment (NadeemShakya) [inline src/views/del-library-page/del-library-page.js:21]: Recommended to leave line after every stylings.

**Review by NadeemShakya — APPROVED** [review]

**Review by nischalshakya95 — APPROVED** [review]

**Pull request merged**